tests/network-tests/src/flows/membership/validatorProfile.ts

```diff
@@ -0,0 +1,30 @@
+import { FlowProps } from '../../Flow'
+import { BuyMembershipHappyCaseFixture } from '../../fixtures/membership'
+
+import { extendDebug } from '../../Debugger'
+import { FixtureRunner } from '../../Fixture'
+import { VerifyValidatorAccountFixture } from '../../fixtures/membership/VerifyValidatorAccountFixture'
+
+export default async function validatorAccount({ api, query }: FlowProps): Promise<void> {
+  const debug = extendDebug('flow:validator-account-update')
+  debug('Started')
+  api.enableDebugTxLogs()
+
+  const VerifyValidator = [
+    {
+      memberId: '1',
+      validatorAccount: 'validator address',
+    },
+    {
+      memberId: '2',
+      validatorAccount: '',
```

Comment:
Please update the parameters here:
```suggestion
      memberId: '1',
      isVerified: true,
    },
    {
      memberId: '2',
      isVerified: true,
      asWorker: {{a membership group worker id}}, // This would be good to make sure non lead worker can call the extrinsic too
```

Also there should be a second call where `isVerified` is set to `false` to make sure workers can reverse the verification if something comes up.